src/cdn/loadCKEditorCloud.ts

```diff
@@ -76,20 +76,20 @@ export function loadCKEditorCloud<Config extends CKEditorCloudConfig>(
 	config: Config
 ): Promise<CKEditorCloudResult<Config>> {
 	const {
-		version, languages, plugins,
+		version, translations, plugins,
 		premium, ckbox
 	} = config;
 
```

Comment:
Instead of creating yet another utility function, can we just filter out the `en` translation here?